west.yml

```diff
@@ -0,0 +1,14 @@
+manifest:
+  self:
+    path: example-application
+
+  remotes:
+    - name: zephyr
+      url-base: https://github.com/zephyrproject-rtos
+
+  projects:
+    - name: zephyr
+      remote: zephyr
+      repo-path: zephyr
```

Comment:
Redundant; `repo-path` defaults to `name`

Reply:
fixed